deb/remote.go

```diff
@@ -302,6 +306,10 @@ func (repo *RemoteRepo) Fetch(d aptly.Downloader, verifier pgp.Verifier) error {
 		// 2. try Release + Release.gpg
 		release, err = http.DownloadTemp(gocontext.TODO(), d, repo.ReleaseURL("Release").String())
 		if err != nil {
+			if err1, ok := err.(*http.Error); ok && (err1.Code == 404 || err1.Code == 403) {
```

Comment:
Same as above:

I’m not sure if this 403 is a good practice. It practically says that there is a Release file, one just is not allowed to access it.